safe/impact_function/multi_exposure_wrapper.py

```diff
@@ -298,6 +321,21 @@ def prepare(self):
             if status != PREPARE_SUCCESS:
                 return status, message
 
+        hazard_name = get_name(self.hazard.keywords.get('hazard'))
+        hazard_geometry_name = get_name(geometry_type(self.hazard))
+        self._name = (
+            u'Multi exposure {hazard_type} {hazard_geometry} On '.format(
```

Comment:
Use title case for `exposure` ? Since `On` is also in title case (although, `on` is preposition). Do we need a translation here?